tests/unit_tests/conftest.py

```diff
@@ -22,6 +25,17 @@ def random_string(length: int = 10, source: str = printable) -> str:
     return ''.join(choice(source) for _ in range(length))
 
 
+def random_int(start: int = 1, end: int = 100) -> int:
+    """
+    Generate a random int in range form start to end
```

Comment:
TYPO: Generate a random int in range **form** start to end

Reply:
Done